src/octoprint/static/js/app/viewmodels/gcode.js

```diff
@@ -566,6 +553,38 @@ $(function() {
         self.onTabChange = function(current, previous) {
             self.tabActive = current == "#gcode";
         };
+        
+        self.incrementLayer = function(value){
+            if (value != self.currentLayer) {
+                event.preventDefault();
+
+                self.layerSlider.slider('setValue', value);
+                value = self.layerSlider.slider('getValue');
+                //This sets the srollbar to the appropriate position.
+                self.layerSlider
+                    .trigger({
+                        type: 'slideStart',
+                        value: value
+                    })
+                    .trigger({
+                        type: 'slide',
+                        value: value
+                    }).trigger({
+                        type: 'slideStop',
+                        value: value
+                    });
+            }
+        };
+        
+        $( "#btn_layer_up" ).click(function() {
```

Comment:
The `click` handler should rather be bound through knockout, so that this stays consistent with other parts of the application.

I suggest to define to methods `incrementLayer` and `decrementLayer` on `self` that do the job, then binding those to the buttons through `data-bind="click: incrementLayer"` and `data-bind="click: decrementLayer"`.